<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>631</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/mesas/gris.jpeg);</string>
   </property>
   <widget class="QPushButton" name="pushMESAM4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string>MESA 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushMESAM1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 0);
border-color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string>MESA 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushMESAM2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string>MESA 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushMESAM5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string>MESA 5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushMESAM6">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string>MESA 6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushMESAM3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string>MESA 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="salirmeseros">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>400</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>salir</string>
    </property>
   </widget>
   <zorder>pushMESAM4</zorder>
   <zorder>pushMESAM2</zorder>
   <zorder>pushMESAM5</zorder>
   <zorder>pushMESAM6</zorder>
   <zorder>pushMESAM3</zorder>
   <zorder>salirmeseros</zorder>
   <zorder>pushMESAM1</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../fr.qrc"/>
 </resources>
 <connections/>
</ui>
